components/org.wso2.carbon.identity.oauth.ui/src/main/resources/org/wso2/carbon/identity/oauth/ui/i18n/Resources.properties

```diff
@@ -97,9 +97,12 @@ invalid.application.access.token.expiry.time=Invalid value for Application Acces
 invalid.refresh.token.expiry.time=Invalid value for Refresh Token Expiry Time
 invalid.id.token.expiry.time=Invalid value for Id Token Expiry Time
 seconds=seconds
-oauth.add.audience=Add
-sp.audience=Audience
-enable.audience.restriction=Enable Audience Restriction
+oauth.add.idtoken.audience=Add
+oauth.add.accesstoken.audience=Add
+sp.idtoken.audience=Audience
+sp.accesstoken.audience=Audience
+enable.idtoken.audience.restriction=Enable ID Token Audience Restriction
+enable.accesstoken.audience.restriction=Enable Access Token Audience Restriction
```

Comment:
better to use `access.token` and `id.token` to be consistent with the other properties defined in this file

Reply:
fixed with d43a24390e23624cc0139558087860229e9dbe7e